_pages/how-to-submit.md

```diff
@@ -0,0 +1,213 @@
+
+### Write and Preview
+
+When completing the issue template, you can add your text in the `Write` tab, and then click on the `Preview` tab to see how your issue is rendered. It can be helpful to read through the issue template in `Preview` mode, as it automatically formats and organizes the information.
+
+One of the quirky things about the pyOpenSci submission template is that the check boxes have two different methods for completion:
+* You can manually add an "X" in the box to mark it as checked,
+* or you can check it off after you submit the issue by clicking on the corresponding boxes.
+
+### Markdown basics
+
+Our issue template is written using Markdown. Because we ask that you not add any additional formatting to the issue template, you don't actually need to know how to use Markdown to complete the issue template! However, knowing the basics of Markdown can be helpful in understanding how the information in the issue template is organized. At a very high level, the Markdown used in the pyOpenSci issue template includes:
+
+* Headers, indicated by `#`. `#` corresponds to an `H1` header, `##` and `H2` header, and so on.
+* Bullet-points, which can be created using `*` or `-`.
+* Checkboxes, which appear as `[ ]`. You can mark a checkbox as completed by adding an "X", like this: `[X]`. The "X" can be uppercase or lowercase.
+* Bolding text, using two asterisks on each side of the text to be bolded, like this: **this text is bold**.
+* Italicizing text, using a single asterisk on each side of the text to be italicized: *this text is italicized*.
+
+> New to Markdown? Here's a great [Markdown Guide](https://www.markdownguide.org/basic-syntax/) that covers all of the basic syntax.
+
+## <i class="fa-solid fa-map"></i> Step-by-step guide to completing the pyOpenSci software review issue template
+
+When completing your package submission, it's best to submit the issue all at once, rather than completing it over the course of several sessions. We recommend reading through the issue once or twice to see what information you'll need, which includes:
+
+* The scope of your package (see categories)
+* Your target audience
+* Scientific applications of your package
+* Other packages that accomplish the same thing (if any), and how yours is different
+
+> Remember: keep the formatting of the issue template exactly as it is! It's OK to leave sections that don't apply to you blank, and there's no need to add any additional formatting (bolding, italicizing, etc.). Any of these types of changes make it more difficult for our editorial team to process your package submission.
+
+### Package metadata
+
+* **Add a title:** keep your title simple. Oftentimes your package name and the word submission is enough, such as "My Amazing Python package submission."
```

Comment:
```suggestion
* **Add a title:** keep your title simple. Oftentimes your package name and the word submission is enough, such as "packageName Submission."
```

you can also provide a link to the softwar-esubmission repo for them to look at other titles. or a screen show of what others have done would work too.